src/app/user/user.ts

```diff
@@ -1,4 +1,14 @@
 export class User {
+  id: string;
   fullName: string;
   imageURL: string;
 }
+
```

Comment:
We can, once they have the same format all over. We are still using the old format for /user API.